lib/logger.js

```diff
@@ -1,5 +1,5 @@
 import { getLogger } from 'appium-logger';
 
-let logger = getLogger('iosCrashLog');
+let logger = getLogger('iOS');
```

Comment:
good point, I forgot this wasn't its own package.

and now it is! looks like I was not forgetting but rather remembering the future :-)